assets/dev/scss/frontend/widgets/link-in-bio.scss

```diff
@@ -0,0 +1,178 @@
+	--e-link-in-bio-ctas-border-radius: 20px;
+	--e-link-in-bio-ctas-border-style: none;
+	--e-link-in-bio-ctas-border-width: 0;
+	--e-link-in-bio-ctas-padding: 20px;
+	--e-link-in-bio-ctas-text-color: inherit;
+
+	margin-inline: auto;
+	max-width: var(--e-link-in-bio-container-max-width);
+	padding: var(--e-link-in-bio-gutter);
+	width: 100%;
+
+	&__content-container {
+		display: flex;
+		flex-direction: column;
+		gap: 1em;
+		margin-inline: auto;
+		max-width: var(--e-link-in-bio-content-max-width);
+		text-align: center;
+
+		* {
+			word-wrap: break-word;
+		}
+	}
+
+	&__identity {
+
+		.e-link-in-bio__identity-image {
+			display: flex;
+		}
+
+		.e-link-in-bio__identity-image-el {
+			display: block;
+			flex: 1 1 100%;
+			object-fit: cover;
+		}
+
+		.e-link-in-bio__identity-image-profile {
+			aspect-ratio: 1;
+			border-color: var(--e-link-in-bio-identity-image-profile-border-color);
+			border-radius: var(--e-link-in-bio-identity-image-profile-border-radius);
+			border-style: var(--e-link-in-bio-identity-image-profile-border-style);
+			border-width: var(--e-link-in-bio-identity-image-profile-border-width);
+			overflow: hidden;
+			margin-inline: auto;
+			max-width: 100%;
+			width: var(--e-link-in-bio-identity-image-profile-width);
+
+			&.has-border {
+				--e-link-in-bio-identity-image-profile-border-style: solid;
+			}
+
+			&.has-style-square {
+				--e-link-in-bio-identity-image-profile-border-radius: 0;
+			}
+
+			.e-link-in-bio__identity-image-el {
+				aspect-ratio: inherit;
+			}
+		}
+
+		.e-link-in-bio__identity-image-cover {
+			border-color: var(--e-link-in-bio-identity-image-cover-border-color);
+			border-style: var(--e-link-in-bio-identity-image-cover-border-style);
+			border-width: 0 0 var(--e-link-in-bio-identity-image-cover-border-bottom-width) 0;
+			margin-block-start: calc(var(--e-link-in-bio-gutter) * -1);
+			margin-inline: calc(var(--e-link-in-bio-gutter) * -1);
+			max-height: var(--e-link-in-bio-identity-image-cover-height);;
+			width: calc(100% + calc(var(--e-link-in-bio-gutter) * 2));
+
+			&.has-border {
+				--e-link-in-bio-identity-image-cover-border-style: solid;
+			}
+		}
+	}
+
+	&__heading {
+		color: var(--e-link-in-bio-heading-color);
+	}
+
+	&__title {
+		color: var(--e-link-in-bio-title-color);
+	}
+
+	&__description {
+		color: var(--e-link-in-bio-description-color);
+	}
+
+	&__icons {
+		display: flex;
+		gap: 1em;
+		justify-content: center;
+	}
+
+	&__icon {
+		color: var(--e-link-in-bio-icon-color);
+		display: flex;
+		height: var(--e-link-in-bio-icon-size);
+
+		&.has-size-medium {
+			--e-link-in-bio-icon-size: 30px;
+		}
+
+		&.has-size-large {
+			--e-link-in-bio-icon-size: 35px;
+		}
+
+		.e-link-in-bio__icon-link {
+			color: inherit;
+			display: flex;
+			flex: 1 1 auto;
+
+			&:hover,
+			&:focus,
+			&:active {
+				color: inherit;
+			}
+		}
+
+		svg {
+			fill: currentColor;
+		}
+	}
+
+	&__ctas {
+		display: grid;
+		grid-template-columns: 100%;
```

Comment:
Use fractions (fr) rather than percentages. This makes more sense when using grid CSS technology.

Reply:
@rodolphebertozzo I agree with you on this one, however I've run into problems with this recently elsewhere where a simple 1fr column can cause layout issue for content in some edge cases so have begun sticking with 100% instead. I'll move it to 1fr for now and keep an eye out.